src/other/broadcasting.jl

```diff
@@ -42,23 +49,23 @@ function Base.copy(bc::Base.Broadcast.Broadcasted{DataFrameStyle})
         throw(ArgumentError("Column names in broadcasted data frames must match. " *
                             "Non matching column names are $msg"))
     end
-    nrows = length(axes(bc)[1])
+    nrows = length(axes(bcf)[1])
```

Comment:
`bcf` doesn't exist?

Reply:
It is defined in the first line of this function. In general we use `bcf` which is a flattened version of `bc` passed later so I changed this for consistency.

The difference between `bc` and `bcf` is that `bcf` is non-nested (it is a function and a list of arguments), while `bc` can be nested.